src/components/layout/modal/uk_account_closure_modal.tsx

```diff
@@ -26,25 +26,25 @@ const UKAccountClosureModal = () => {
             onPositive={handlePositive}
             is_dismissible={false}
         >
-            <Header type="paragraph-2" weight="regular">
+            <Header as="h1" type="paragraph-2" weight="regular">
                 <Localize translate_text="As part of the changes in our market offerings, we will be closing our UK clients’ accounts." />
             </Header>
-            <Header type="paragraph-2" mt="1.6rem">
+            <Header as="h1" type="paragraph-2" mt="1.6rem">
                 <Localize translate_text="What this means for you" />
             </Header>
-            <Header type="paragraph-2" weight="regular" mt="0.8rem">
+            <Header as="h1" type="paragraph-2" weight="regular" mt="0.8rem">
                 <Localize translate_text="You can no longer trade on Deriv or deposit funds into your account." />
             </Header>
-            <Header type="paragraph-2" weight="regular" mt="0.8rem">
+            <Header as="h1" type="paragraph-2" weight="regular" mt="0.8rem">
                 <Localize translate_text="If you had any open positions, we have closed them and refunded you." />
             </Header>
-            <Header type="paragraph-2" mt="1.6rem">
+            <Header as="h1" type="paragraph-2" mt="1.6rem">
                 <Localize translate_text="What you need to do now" />
             </Header>
-            <Header type="paragraph-2" weight="regular" mt="0.8rem">
+            <Header as="h1" type="paragraph-2" weight="regular" mt="0.8rem">
                 <Localize translate_text="Please proceed to withdraw all your funds from your account." />
             </Header>
-            <Header type="paragraph-2" weight="regular" mt="0.8rem">
+            <Header as="h1" type="paragraph-2" weight="regular" mt="0.8rem">
```

Comment:
Why we need to add h1 to all of these tags
this much h1 is not allowed for SEO
in each section we can have just one h1 tag

Reply:
@ali-hosseini-fs  it's fixed

Reply:
thanks @iman-fs, well done
we have a problem with typography component that we have to fix it in another PR
not related to your changes.